.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and Test
+
+on:
```

Comment:
I just don't quite understand why this is in the PR... this code is already in main -__- git seems confused about something here I think.